### ISSUE TITLE
placement of "sperm motility"

### ISSUE THREAD

**Issue by gocentral** [geneontology/go-ontology#10905]
Hi,

I am looking at "sperm motility", which is an is_a child of "cell motility".

I think "sperm motility" should be moved down to be an is_a child of "ciliary cell motility" since the sperm flagellum is a specialized type of cilium. Then, the "regulation of sperm motility" terms should be be made types of the more general "regulation of ciliary cell motility" terms.

I also see that there is a term for "cilium movement involved in cell motility", but I don't see a parallel term for "sperm flagellum movement involved in sperm motility". It seems potentially useful given that there are a number of specialized proteins present in sperm flagella, but not in other cilia that contribute to the extreme needs of the sperm flagellum. Of course, it might be worth considering the relations to terms like "sperm capacitation" as well before making changes based on this last suggestion.

thanks,

-Karen

Reported by: krchristie

Original Ticket: [geneontology/ontology-requests/10719](https://sourceforge.net/p/geneontology/ontology-requests/10719)

**Comment by krchristie** [comment]
In an email discussion, Paola has pointed out that the cross-product definition of “sperm motility” is to a CL term that is a SubClassOf “ciliated cell”. Thus the logical definition of the term “sperm motility” is inconsistent with its position in the ontology. 
http://bioportal.bioontology.org/ontologies/CL?p=classes&conceptid=CL%3A0000019

As David H said earlier, C. elegans has amoeboid sperm, so within actively annotated model organisms, we have two types of sperm motility:
- amoeboid sperm motility
- flagellated sperm motility

The existing logical definition would be correct for "flagellated sperm motility" and such a term looks like it should be a child of this term: "cilium-dependent cell motility (GO:0060285)”. 

In the recent email discussion involving John Van Dam, it was noted that the sperm of angiosperms are not flagellated. However, from Dresselhaus et al. 2016. Fertilization Mechanisms in Flowering Plants (PMID:26859271), it looks like the sperm of angiosperms are not considered to be motile cells, so possibly don't need a type of "sperm motility" term.

-Karen

@paolaroncaglia 
@JohnvanDam
@tberardini

**Comment by tberardini** [comment]
Yep, no motile sperms in angiosperms.  

However, gymnosperms have flagellated and motile sperm cells.  http://www.ncbi.nlm.nih.gov/pubmed/16736257

**Comment by krchristie** [comment]
Thanks Tanya, I think that the flagellated motile sperm of gymnosperms would be covered by a general term for "flagellated sperm motility" though. I'm not aware of anything that would necessitate a special term here.

**Comment by tberardini** [comment]
Yes, I thought that a 'flagellated sperm motility' term would work for gymnosperms too.  Cool biology, eh?

**Comment by krchristie** [comment]
Hi Paola,

While you're working on cilia GH tickets, would you see if you can take a look and possibly finish this one too. thanks,

-Karen

@paolaroncaglia

**Comment by paolaroncaglia** [comment]
Hi @krchristie ,

Yes, I'm hoping to work more on ciliary processes this week, and the motility branch is in my to-do list :-)

Cheers,

Paola.

**Comment by krchristie** [comment]
Thanks @paolaroncaglia !!

**Comment by paolaroncaglia** [comment]
https://github.com/geneontology/go-ontology/issues/10898

**Comment by paolaroncaglia** [comment]
I have reviewed this thread. Looking at things more closely, CL:0000019 ’sperm’ is a descendant of ‘animal cell’, see http://www.ontobee.org/ontology/CL?iri=http://purl.obolibrary.org/obo/CL_0000019. So, even though CL:0000019 ’sperm’ is not formally taxon-restricted in CL (…neither is ‘animal cell’!), I think it’s safe to say that CL:0000019 ’sperm’ refers strictly to animal sperm. Not to plant sperms, so that takes care of the whole angiosperms/gymnosperms stuff. :-) As for animals that have sperms that are not ciliated, such as C. elegans, that’s also taken care of, because CL:0000019 ’sperm’ is_a ‘ciliated cell’, which worm sperms clearly aren’t. (Note that ‘ciliated cell’ is formally linked to GO ‘cilium’.) Therefore, any GO term that cross-references CL:0000019 ’sperm’, including ‘sperm motility’, cannot be used for plants, nor for any non-plant species that doesn’t have ciliated sperms.

This said, the pending issues are:

- How can we make the above more explicit in GO?
In a previous email, Karen noted: “It seems clear to me that the existing term “sperm motility” is meant to refer to ciliated/flagellated sperm [correct], so we could change the name from “sperm motility” to “ciliated sperm motility”, and move it to be a child of this term: "cilium-dependent cell motility (GO:0060285)”.”. It seems that all ‘sperm’ terms in GO clearly refer to the same animal, ciliated sperm; ‘sperm part’ does so even formally, via part_of. Therefore, for consistency, I would not change the primary GO names of sperm-related classes, but I’d certainly add an exact synonym “ciliated sperm motility” to “sperm motility”. I will also edit its definition from “Any process involved in the controlled movement of a sperm cell.” to “Any process involved in the controlled movement of a ciliated animal sperm cell.”.
As for the placement, yes, I will move “sperm motility” under "cilium-dependent cell motility”. 
I will also add a taxon rule: sperm motility only_in Metazoa. 
Any objections?

- It appears that current annotations to ‘sperm motility’ and descendants cover Caenorhabditis (haven’t checked other species yet). Those will need to move elsewhere. I’ll need to look into this further. 

- In her initial request, Karen asked for "sperm flagellum movement involved in sperm motility”. To this aim:
a) I’d create ‘sperm flagellum movement’ (is_a cilium movement, and mediated_by sperm flagellum - or perhaps has_participant or occurs_in - see https://github.com/geneontology/go-ontology/issues/10898)
b) Karen could request "sperm flagellum movement involved in sperm motility” via TG template.

- Longer-term: we may want to look at ‘cell motility’ and revamp the entire branch based (more) on the different mechanisms of motility. See my (Chris’) comment on May 9, 2014.

**Comment by ukemi** [comment]
I'm not sure I am completely comfortable with this. I believe some crustaceans also have sperm without flagella. It seems to me the CL term is too restrictive.

**Comment by paolaroncaglia** [comment]
Argh...

**Comment by paolaroncaglia** [comment]
@ukemi 
We could certainly suggest to CL that they add the missing term, as long as we can provide them PMIDs or ISBNs to support it. I was just saying that the GO terms referencing CL 'sperm' actually mean 'ciliated animal sperm' blahblah.

**Comment by krchristie** [comment]
Hi Paola,

I agree with David that it might be good if we could use a CL term that is less restrictive to Metazoa, but not because of crustaceans that have sperm without flagella, but because of gymnosperms that do have flagellated sperm. If it is possible to use a more general term that would allow for any flagellated/ciliated cell, I think that would be better. Basically, I think that the CL term xref	for "flagellated sperm motility" should	be to a term with as broad a taxonomic range as possible since the presence of cilia/flagella seems to be very ancient. However, if the CL doesn't go to a level that makes that possible, I am OK with a less general term, though at some point	in the future if GO expands its	mandate	for organismal coverage, we might	need to use a more general term.

I am also concerned about having this term mean "flagellated sperm motility", but NOT changing its name to say that since there are other kinds of sperm motility that do not involve flagella. I think this will lead to annotators using this term to annotate other kinds of sperm motility.  I think we need a general term for "sperm motility" and then under it terms for sperm motility that depend on different mechanisms, e.g. "flagellated sperm motility", "amoeboid sperm motility", perhaps others.	

Basically, I think we need something like this:
- sperm	motility
-- flagellated sperm motility
-- amoeboid sperm motility

It seems to me that the existing term for "sperm motility" refers to "flagellated sperm motility, based on its existing cross products. However, whether it is better for the existing term "sperm motility" to remain the top level sperm (to avoid making any current annotations incorrect) or to make it more specific based on its current cross-products is an open question that I don't feel strongly about.

-Karen

**Comment by vanaukenk** [comment]
@krchristie suggestion sounds reasonable to me.  I agree that a more general 'sperm motility' parent term with child terms referring to the different mechanisms might be best.
It also seems that there may be some work to do on the CL wrt the different types of sperm and getting the parentage correct?

**Comment by ukemi** [comment]
I like Karen's plan as well.

**Comment by paolaroncaglia** [comment]
Thanks all. Then this becomes a question for CL first of all... I'm not sure what their bandwidth is at the moment, and what timeline this would imply. @cmungall , any idea? I can certainly create a ticket on their tracker, unless one of you would like to give it a shot.

**Comment by krchristie** [comment]
Do proposed changes in GO really depend on CL? It seems that we could make the changes I'm proposing in GO without waiting for CL, using the existing CL reference for the xref of the "flagellated sperm motility" term for now and changing that later if/when CL provides a more general term.

**Comment by paolaroncaglia** [comment]
That could do too, yes.
I'll come back to this hopefully tomorrow.

**Comment by cmungall** [comment]
It sounds like there should be a ticket with two items should be opened in the CL tracker:

 * remove assumption that all sperm is ciliated
 * remove assumption that all sperm is animal

**Comment by paolaroncaglia** [comment]
To start with, I opened a ticket in the CL tracker, feel free to subscribe:
https://github.com/obophenotype/cell-ontology/issues/437

**Comment by paolaroncaglia** [comment]
Back to Karen’s comment

“I think we need a general term for "sperm motility" and then under it terms for sperm motility that depend on different mechanisms, e.g. "flagellated sperm motility", "amoeboid sperm motility", perhaps others.
Basically, I think we need something like this:
- sperm motility
-- flagellated sperm motility
-- amoeboid sperm motility
It seems to me that the existing term for "sperm motility" refers to "flagellated sperm motility, based on its existing cross products. However, whether it is better for the existing term "sperm motility" to remain the top level sperm (to avoid making any current annotations incorrect) or to make it more specific based on its current cross-products is an open question that I don't feel strongly about.”

Thinking about the best strategy here, I’m looking at the manual experimental annotations to ‘sperm motility’ and its descendants, including regulation terms. There are 139, to 9 species:

Mus musculus		
Drosophila melanogaster		
Homo sapiens		
Caenorhabditis elegans		
Rattus norvegicus		
Danio rerio		
Coturnix coturnix japonica		
Sus scrofa		
Lytechinus variegatus

All have flagellated sperms *except* Caenorhabditis, which accounts for 7 annotations (6 WormBase + 1 UniProt).
Therefore, I will 

- rename and redefine the existing ’sperm motility’ as ‘flagellated sperm motility’ and retain its cross-reference to CL:0000019 ’sperm’

- rename and redefine the existing triad of regulation terms for ’sperm motility’ accordingly

- create a broad parent for ‘flagellated sperm motility’ (’sperm motility’) and a sibling (‘amoeboid sperm motility’)

- add regulation terms for ‘amoeboid sperm motility’ (reg + pos reg only)

- when ready, ask WormBase and UniProt to move Caenorhabditis annotations to the corresponding amoeboid terms (including regulation terms)

- create ‘sperm flagellum movement’ (is_a cilium movement, and mediated_by sperm flagellum - or perhaps has_participant or occurs_in - see #10898). Karen could then request "sperm flagellum movement involved in flagellated sperm motility” via TG template.

- if and when CL revises their ‘sperm’ branch, we may adjust/add cross-references to ‘sperm motility’ terms.  

- Longer-term: we may want to look at ‘cell motility’ and revamp the entire branch based (more) on the different mechanisms of motility. See my (Chris’) comment on May 9, 2014.

**Comment by paolaroncaglia** [comment]
Hi @vanaukenk,

(Please refer to the checklist in my previous comment for a summary)
There are currently 6 WormBase annotations to ‘sperm motility’ and regulation thereof. (Note the terms have just been renamed to ‘flagellated sperm motility’ etc. but this name change may not have percolated yet.) Could you please rehouse those annotations to the new ‘amoeboid sperm motility’ terms below as appropriate?

GO:0097723 amoeboid sperm motility
GO:1905416 regulation of amoeboid sperm motility
GO:1905417 negative regulation of amoeboid sperm motility
GO:1905418 positive regulation of amoeboid sperm motility

Thanks,

Paola

**Comment by paolaroncaglia** [comment]
Note for self: emailed GOA about the single UniProt annotation, here's my email text for reference:

"Could you please move the following annotation
UniProtKB    P34370    cil-1        GO:1902093    PMID:19781942 IMP        P    Inositol polyphosphate 5-phosphatase I5P2_CAEEL|cil-1|C50C3.7    protein    taxon:6239    20130712 UniProt
from the current term GO:1902093 'positive regulation of sperm motility'
to the new term GO:1905418 'positive regulation of amoeboid sperm motility'?"

**Comment by paolaroncaglia** [comment]
Hi @krchristie ,

I just added GO:0097724 ‘sperm flagellum movement’ (is_a cilium movement), and renamed GO:0030317 as ‘flagellated sperm motility’.
If you wish, you may now request ‘sperm flagellum movement involved in flagellated sperm motility’ via TG template. Thanks.

**Comment by paolaroncaglia** [comment]
Hi @cmungall and/or @dosumis, 

(You may ignore the rest of this ticket)
Would it be appropriate to add the following links please:

cilium movement has_participant cilium
sperm flagellum movement has_participant sperm flagellum
cilium-dependent cell motility has_participant ciliated cell

Or would occurs_in or mediated_by be more correct?
Thanks!

**Comment by paolaroncaglia** [comment]
Note for self:
The action items left to do for this ticket (see unchecked boxes in my comment dated June 23rd 2016) are of minor priority, so I’ll set this ticket to pending.

**Comment by krchristie** [comment]
Thanks very much Paola for your work in resolving this issue. I've requested the ‘sperm flagellum movement involved in flagellated sperm motility’ term via TG template as you suggested.

thanks!!

-Karen

**Comment by paolaroncaglia** [comment]
Note for self: Alice emailed me this morning:
"I tried to correct the annotation as you requested, but it looks like the changes have not be implemented in QuickGO and it does not recognize the new term. I'll try next week again."

**Comment by paolaroncaglia** [comment]
Hi @cmungall  and @dosumis ,

May I please double-check something with you. For the cilia-GO project, I’m looking into creating links between cilium movement BP terms and ciliary CC terms/ciliated cell CL terms. In particular between

1) cilium movement and cilium
2) sperm flagellum movement and sperm flagellum

I’m leaning towards using has_agent CC, e.g. cilium movement has_agent cilium.
occurs_in CC seems reductive and incomplete (e.g. cilium movement occurs_in cilium), but would propagate annotations.
has_agent cilium/sperm flagellum AND occurs_in CL:0000064 ciliated cell/CL 0000019 sperm seems more complete, but CL is currently pointing to animal cells at the moment and not considering plants. So this option may create TPV currently and until CL expands to new taxa and retains the general classes (see https://github.com/obophenotype/cell-ontology/issues/437). Trying to create a link at a lower level between cilium-dependent cell motility and ciliated cell would also incur in the same problem.

Should I just add cilium movement/sperm flagellum movement has_agent cilium/sperm flagellum and stop there for now?

Thanks!

**Comment by dosumis** [comment]
has_agent* is not wrong, so I don’t see a problem with adding.  We already have links in the other direction - via capable_of_part_of (this should probably be changed to capable_of).

I think we need to ditch the CL link for now as will remain too taxon restricted for the foreseeable future.


* Chris:  we have some cleanup work to do here, merging with ‘mediated_by’.  Official label in RO not has_agent - revisit?

**Comment by cmungall** [comment]
On 6 Oct 2016, at 1:37, David Osumi-Sutherland wrote:

> * Chris: we have some cleanup work to do here, merging with 
> ‘mediated_by’. Official label in RO not has_agent - revisit?

+1

**Comment by cmungall** [comment]
Is has_agent correct in the general case here?

Maybe this doesn't work for cilia, but think of a person riding a bicycle. The agent in the bicycle-movement process is the person, not the bicycle or the mereological sum of person+bike.

In this case the bicycle undergoes movement, and the movement is undergone_by bicycle. We don't have such a relation in RO, but there are a number of cases where this would be useful.

**Comment by paolaroncaglia** [comment]
Hi @cmungall and @dosumis,

I agree with Chris - the real agent in the movement of the cilium is one of its parts and not the whole cilium (the molecular motor, which has ‘effectors’, again ciliary parts). So none of the available solutions look satisfactory at the moment, and it’s not a priority, so I’ll drop this. 

David suggests changing
motile cilium capable of part of cilium movement
to
motile cilium capable of cilium movement
which sounds correct for this particular case, but I thought the use of capable of was reserved to MFs?

**Comment by dosumis** [comment]
> but I thought the use of capable of was reserved to MFs?

Nope.  If a structure is capable of carrying out a whole BP or MF then capable_of applies.  If it only capable of carry out a part then capable_of_part_of applies. (The relations are not GO specific: In VFB we use capable_of an capable_of_part_of to record the functions of neurons.)

**Comment by paolaroncaglia** [comment]
Thanks @dosumis , I updated that link.

**Comment by paolaroncaglia** [comment]
Only action item left: 
- [ ] Longer-term: we may want to look at ‘cell motility’ and revamp the entire branch based (more) on the different mechanisms of motility. See this comment: 
https://github.com/geneontology/go-ontology/issues/10905#issuecomment-124912611

**Comment by paolaroncaglia** [comment]
Now moved to https://github.com/geneontology/go-ontology/issues/12722. So, closing this one.
